src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      this.result += value;
+    },
+    subtract(value) {
+      this.result -= value;
+    },
+    multiply(value) {
+      this.result *= value;
+    },
+    divide(value) {
+      this.result /= value;
```

Comment:
Consider adding a check to prevent division by zero in the `divide` method. This can prevent potential runtime errors or unexpected behavior.